starter-code/views/beerCard.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="mainContainer">
+  <div class="beerContainer">
+    <img class="beerImg" src="{{this.image_url}}" alt="">
```

Comment:
Aca recuerden que si hacemos referencia a this como objeto, podemos prescindir de el y solo pooner las propiedades.